apps/organizations/constants/defaults.py

```diff
@@ -27,3 +27,8 @@
     'user__last_name',
     'user__username',
 )
+
+HTK_ORGANIZATION_INVITATION_RESPONSE_URL_NAME = ''
+HTK_ORGANIZATION_INVITATION_EMAIL_TEMPLATE_NAME = ''
+HTK_ORGANIZATION_INVITATION_EMAIL_SUBJECT = 'You have been invited to join {}'
+HTK_ORGANIZATION_INVITATION_AUTO_ADD_TO_WAIT_LIST = True
```

Comment:
I think this should just always be true, unless you think otherwise?

And if it's always true, let's just remove the setting and condition in the code.

Reply:
Removed